src/index.ts

```diff
@@ -184,6 +186,185 @@ export function HTPipeAttachData<
   }
 }
 
```

Comment:
why the | before boolean?  Is that even legal?
- [ ] nothing or something?

Reply:
that was added by prettier
i checked and just in case i create a test to check the type of HTPipePreAuthorize and there is no difference when we have | before boolean or we don't have it

Reply:
wow weird.  ok